cmd/ddev/cmd/debug-refresh.go

```diff
@@ -12,12 +13,17 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	buildAll bool
+	service  string
+)
+
 // DebugRefreshCmd implements the ddev debug refresh command
 var DebugRefreshCmd = &cobra.Command{
 	ValidArgsFunction: ddevapp.GetProjectNamesFunc("all", 1),
```

Comment:
`refresh` was a great name for this when it was created, but how about changing it to `rebuild` with `refresh` being an alias?

Reply:
Okay, `rebuild` gives more context, I'll change it.